### ISSUE TITLE
Gnome Extensions - No property group on GtkToggleButton

### ISSUE THREAD

**Issue by markmcnaughton** [wotmshuaisi/goldpricemonitor#1]
Good day,

When using Gnome Extensions 3.38.4 I see the following error when opening the extensions settings. I'm running Manjaro Gnome with all updates installed.

The settings of extension Gold_Price_Monitor@wotmshuaisi_github had an error:
```
Error: No property group on GtkToggleButton
```

Stack trace:
```
_init/Gtk.Widget.prototype._init@resource:///org/gnome/gjs/modules/core/overrides/Gtk.js:45:40
buildPrefsWidget/<@/home/mark/.local/share/gnome-shell/extensions/Gold_Price_Monitor@wotmshuaisi_github/prefs.js:59:22
buildPrefsWidget@/home/mark/.local/share/gnome-shell/extensions/Gold_Price_Monitor@wotmshuaisi_github/prefs.js:50:45
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
start@resource:///org/gnome/gjs/modules/script/package.js:190:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**Comment by wotmshuaisi** [comment]
Hello @markmcnaughton,

I recently updated this extension to make it compatible with Gnome 40. and according to Gtk API reference property `group` of GtkToggleButton is only available on Gtk 4.

I updated metadata to make sure gnome 3 users can avoid this issue. follow next steps:

1. Uninstall `Gold Price Monitor` extension
2. Download `Gold Price Monitor` extension (shell version 3.36, extension version: 16)
3. `mkdir -p ~/.local/share/gnome-shell/extensions/Gold_Price_Monitor@wotmshuaisi_github`
3. Decompress zip file into folder `~/.local/share/gnome-shell/extensions/Gold_Price_Monitor@wotmshuaisi_github`
4. If you are running Xorg Gnome, alt+f2, type `r` to restart gnome session
5. Logout and login in again if you are running Wayland Gnome

Once my new update got approval, you won't receive any Gnome 40 related updates as a gnome 3 user.

References:

> [https://developer.gnome.org/gtk3/stable/GtkToggleButton.html](https://developer.gnome.org/gtk3/stable/GtkToggleButton.html)
> [https://developer.gnome.org/gtk4/stable/GtkToggleButton.html](https://developer.gnome.org/gtk4/stable/GtkToggleButton.html)
